modules/keycloak/templates/keycloak.json.tpl

```diff
@@ -34,6 +34,10 @@
       }
     ],
     "environment": [
+      {
+        "name" : "ENVIRONMENT",
```

Comment:
Will this still be necessary if we can select the right configuration at build time in Jenkins?

I'm always a bit wary of adding an "environment" parameter to an application because the conditional logic within the application tends to get messy, and you end up with an application that contains configuration for other environments (even though it's never used). It's easier to follow the application logic if you can inject the environment-specific configuration instead, like we're doing for the frontend URL just below.

It's just a slight concern. 🙂 If it's necessary then it's fine - it would just be good to consider alternatives.